src/OpenTelemetry.Instrumentation.AspNetCore/Implementation/AspNetCoreInstrumentationEventSource.cs

```diff
@@ -43,7 +43,7 @@ public void NullPayload(string handlerName, string eventName)
             this.WriteEvent(1, handlerName, eventName);
         }
 
-        [Event(2, Message = "Request is filtered out.", Level = EventLevel.Verbose)]
+        [Event(2, Message = "Request is filtered out. EventName: '{0}'.", Level = EventLevel.Verbose)]
```

Comment:
Would this not cause an allocation?

Reply:
I'm not an expert in EventSource but I don't think so. We pass a constant into `RequestIsFilteredOut` and if no one is listening it will just no-op. If someone is listening I think this template is passed as-is (it isn't actually formatted).

I view this change more as a bug fix. If you look at the other events across the repo, I don't think there are any where something is passed as event args and NOT part of the message template 🤣

Reply:
> I view this change more as a bug fix. If you look at the other events across the repo, I don't think there are any where something is passed as event args and NOT part of the message template 🤣

Yeah. But if there's no listener, we would still call the `WriteEvent` method. We usually do the check for listener in a `NonEvent` method.

Reply:
The `NonEvent` way is used mostly when we have to allocate a string for `ex.ToInvariantString()`. So, I think even though `RequestIsFilteredOut` will call `WriteEvent` method, it would not have to allocate new string for replacing the placeholders in the template.